PinpointKit/PinpointKit/Sources/Fonts.swift

```diff
@@ -32,7 +32,7 @@ public extension UIFont {
      
      - returns: A Source Sans Pro `UIFont` at the specified size and weight.
      */
-    public static func sourceSansProFontOfSize(_ fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
+    public static func sourceSansProFont(ofSize fontSize: CGFloat, weight: FontWeight = .regular) -> UIFont {
```

Comment:
So just to be clear, you’re thinking:

```swift
public static func sourceSansPro(ofSize: size CGFloat, weight: FontWeight = .regular) -> UIFont
public static func menloRegular(ofSize size: CGFloat) -> UIFont
```

With call sites looking like:

```swift
UIFont.sourceSansPro(ofSize: 32.0)
UIFont.menloRegular(ofSize: 16.0)
```

I’d be happy to make the change but am worried we’re deviating from the naming changes in `UIFont`. `UIFont` APIs _were_ renamed, from e.g.:

```swift
public class func systemFontOfSize(fontSize: CGFloat, weight: CGFloat) -> UIFont
```

to:

```swift
public class func systemFont(ofSize fontSize: CGFloat, weight: CGFloat) -> UIFont
```

with no dropping of "font".

At this stage, would a radar be more appropriate (perhaps we can call out specifics in the API guidelines to warrant these APIs changing), or do you think it’s not worth keeping directly in sync with similar signatures in `UIKit`?